src/views/AddItem.jsx

```diff
@@ -61,8 +80,7 @@ export function AddItem({ listToken }) {
 				<input type="submit" value="Add item" />
 			</form>
 			{/* TODO: we could change item added message to a toast message, alert, timeout or use third-party library for this message. */}
```

Comment:
Is this TODO still valid? If so, this seems like a good time to address it.

Reply:
We thought that this might end up being a team decision towards the end, since we may want to make not only alerts on this page, but all alerts consistent across the app.